src/Common/src/System/Security/Cryptography/DerEncoder.cs

```diff
@@ -353,6 +353,46 @@ internal static byte[][] SegmentedEncodeNamedBitList(byte[] bigEndianBytes, int
         }
 
         /// <summary>
+        /// Encodes an octet string using distinguished encoding rules
+        /// </summary>
+        /// <param name="data">Octet string as a byte array</param>
+        /// <returns>Der representation of the octet string</returns>
+        internal static byte[] EncodeOctetString(byte[] data)
+        {
+            return DerEncoder.ConcatenateArrays(SegmentedEncodeOctetString(data));
+        }
+
+        /// <summary>
+        /// Encodes an DateTime object using distinguished encoding rules
+        /// </summary>
+        /// <param name="data">DateTime object to be encoded in utc format</param>
+        /// <returns>Der representation of the utc time</returns>
+        internal static byte[] EncodeUtcTime(DateTime utcTime)
+        {
+            return DerEncoder.ConcatenateArrays(SegmentedEncodeUtcTime(utcTime));
+        }
+
+        internal static byte[][] SegmentedEncodeUtcTime(DateTime utcTime)
+        {
+            Debug.Assert(utcTime != null);
```

Comment:
> I'm sort of surprised that's not a compile warning/error

Yeah.  I seem to remember there being some kind of compat issue where with Roslyn it was made into a warning or an error but then that was removed because it ended up breaking too many projects.  Or something like that.